_drafts/restic-foundation1-cdc.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: post
+title: Foundation - Introducing Content Defined Chunking (CDC)
+---
+
+This post will explain Content Defined Chunking (CDC) and how it is used by
+restic.
+
+Backup programs need to deal with large volumes of changing data. Saving the
+whole copy of each file again to the backup location when a subsequent (usually
+called "incremental") backup is created is not efficient. Over time, different
+strategies have emerged to handle data in such a case.
+
+In a backup program, data de-duplication can be applied in two locations:
+Removing duplicate data from the same or different files within the same backup
+process (*inter-file de-duplication*), e.g. the initial backup, or removing it
+between several backups of the same data (*inter-backup de-duplication*). While
```

Comment:
maybe "several backups that contain some of the same data" instead of "several backups of the same data"? otherwise it sounds like I'm doing a second backup of exactly the same data... why would I do that :)